src/hooks/usePollForGetLeaderboard.ts

```diff
@@ -0,0 +1,34 @@
+import { useAppDispatch, useAppSelector } from "../state/reduxHooks";
+import axios from "axios";
+import { useEffect } from "react";
+import { API_URL } from "../settings";
+import { setLeaderboard, setRequestUpdatedLeaderboard } from "../state/leaderboardSlice";
+
+export function usePollForGetLeaderboard() {
```

Comment:
We can remove "PollFor" from the name because we aren't polling continuously, we're just sending one request

Reply:
done